forms-web-app/src/views/appellant-submission/upload-application.njk

```diff
@@ -0,0 +1,79 @@
+{% extends "layouts/main.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+
+{% block pageTitle %}
+  Original planning application | Appeal a householder planning decision
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {% if errorSummary %}
+        {{ govukErrorSummary({
+          titleText: "There is a problem",
+          errorList: errorSummary
+        }) }}
+      {% endif %}
+
+      <form action="" method="post" novalidate enctype="multipart/form-data">
+        <span class="govuk-caption-l"><span class="govuk-visually-hidden">Section. </span>Required documents</span>
+        <h1 class="govuk-heading-l">
+          Upload the planning application form
+        </h1>
+        <p>If you do not have your original planning application form, you can find it by searching for your planning application on your local planning department's website.</p>
+
+        {{ govukInsetText({
+          text: "File size should be no more than 1.21GB"
+        }) }}
+
+        {% if appeal['upload-application'] %}
+          <h3 class="govuk-heading-m">File added:</h3>
+          <ul class="govuk-list">
+            <li id="appeal-statement-file-name">{{ appeal['upload-application'].name }}</li>
+          </ul>
+        {% endif %}
+
+        <div class="govuk-form-group">
+          <label class="govuk-heading-m" for="upload-application">
+            Upload a file
+          </label>
+          {% if errors['upload-application'] %}
+            <span id="upload-application-error" class="govuk-error-message">
+            <span class="govuk-visually-hidden">Error:</span> {{ errors['upload-application'].msg }}
+          </span>
+          {% endif %}
+          <input class="govuk-file-upload" id="upload-application" name="upload-application" type="file">
```

Comment:
Just curiosity, what was wrong with the govuk input file ?

Reply:
Right yeah, good spot. 

Nothing is wrong with it. 

The mistake I have made is that with the previous PR, the prototype called for a very specific CSS class being added to the label for the file upload component. That class conflicted with the one that is mandatory on the gov uk file upload nunjucks template. 

What I've done here is copy / paste from the other template, and in this instance, I don't need the class so should have used the component. Good spot.

Reply:
Will do this as a separate PR

Reply:
Disregard - this page has the exact same issue - they want a custom style on the "upload a file" text which isn't possible with nunjucks

![Screenshot from 2020-11-25 08-17-06](https://user-images.githubusercontent.com/6329541/100200509-a4ffeb80-2ef6-11eb-9101-0580c46c9d46.png)